lib/src/complex.dart

```diff
@@ -116,20 +116,20 @@ class Complex {
   /// but at least one part is infinite.
   double abs() {
     if (isNaN) {
-      return double.NAN;
+      return double.nan;
     }
     if (isInfinite) {
-      return double.INFINITY;
+      return double.infinity;
     }
     if (_real.abs() < _imaginary.abs()) {
-      if (_imaginary == 0.0) {
```

Comment:
Variables `_real` and `_imaginary` have been switched several time in `abs()`.

Reply:
The original code was:
```
if(_real.abs() < _imaginary.abs()){
  if(_imaginary == 0.0){
    return _real.abs()
  }
  ...
}
```

That has no sense, because if `_imaginary` is equal to zero implies `_real.abs() < 0.0`, and it is impossible. the same in the other case.

Reply:
Hello. If I am wrong, let me know which is my mistake to resend the pull request.